.maintain/chaostest/README.md

```diff
@@ -0,0 +1,89 @@
+chaostest
+=========
+
+A cli for chaos testing on substrate
+
+[![oclif](https://img.shields.io/badge/cli-oclif-brightgreen.svg)](https://oclif.io)
+[![Version](https://img.shields.io/npm/v/chaostest.svg)](https://npmjs.org/package/chaostest)
+[![Downloads/week](https://img.shields.io/npm/dw/chaostest.svg)](https://npmjs.org/package/chaostest)
+
+<!-- toc -->
+* [Usage](#usage)
+* [Commands](#commands)
+<!-- tocstop -->
+# Usage
+<!-- usage -->
+```sh-session
+$ npm install -g chaostest
```

Comment:
We're using `yarn` in all other projects, we should stick to that here for consistency.

Reply:
NPM generates a correct lock file whereas a Yarn lock file could be corrupt in some cases and has to be fixed with yarn-tools. 
(This step is also taken in CI runners)